Assets/Scripts/PlayerController.cs

```diff
@@ -136,68 +110,60 @@ public class PlayerController : MonoBehaviour {
 			break;
 		case 4:
 			playerColor = "D97A7B";
-			break;
-		}
-
-		if (playerType == 0) {
+			break;
+        }
+
+        // Map shape numbers to names, order is important
+        string[] shapeNames = new string[] {

```

Comment:
I wonder if there's way we can use enums to get around this sort of thing -- https://unity3d.com/learn/tutorials/topics/scripting/enumerations